hyperspy/tests/misc/test_cython.py

```diff
@@ -0,0 +1,24 @@
+import nose.tools as nt
+import warnings
+
+import os
+
+from nose.plugins.skip import SkipTest
+
+my_path = os.path.dirname(__file__)
+
+def test_cythonization():
+    try:
+        import Cython
+    except ImportError:
+        raise SkipTest
+    
+    if not os.path.exists(my_path + '/cython/test_cython_integration.c'):
+        raise RuntimeError('deprecation of cythonization of pyx extensions detected!')
```

Comment:
Maybe the text should be something like "Cythonization of <file path> extension failed"?